### ISSUE TITLE
JAWS + FF: Input + Datalist has focus issue & inadequate announcments compared to JAWS + Chrome

### ISSUE THREAD

**Issue by scottaohara** [FreedomScientific/VFO-standards-support#11]
### Summary
When interacting with a text input with a datalist, JAWS announces the input as having a popup, but with no other announcements on how to interact with the popup.  When hitting the down arrow to attempt to interact with the datalist, the datalist is opened, but JAWS instead moves focus to the next element in the DOM.

If using JAWS + Chrome, the input is announced as a combo box and JAWS states "to change the selections, use the arrow keys".

### Expected result
On pressing down arrow, it should be expected that the first item of the datalist would receive focus, as is the case when JAWS is not running.  

Chrome + JAWS works as expected.

### Actual result
When hitting the down arrow, the datalist opens, but JAWS instead moves focus to the next element in the DOM. In the test page, that element is an `<h3>` with text "code". If the up arrow or SHIFT + Tab are used to return JAWS focus to the input, the down arrow key can now be used to enter the datalist options.

### Example
https://thepaciellogroup.github.io/AT-browser-tests/test-files/datalist.html

#### JAWS version and build number
JAWS 18.0.4321
[Issue previously reported pertaining to JAWS 16](https://github.com/ThePacielloGroup/AT-browser-tests/issues/22)
 
#### Operating System and version
Windows 10 Pro v.1703 64-bit

#### Browser and version:
Firefox 56.0
Chrome 61.0.3163

**Comment by stevefaulkner** [comment]
@scottaohara i think this may be a Firefox bug as Firefox has not implemented input type=text + datalist as a combobox, which is what is defined in the HTML acc API spec https://w3c.github.io/html-aam/#details-id-75 from what you have described it sounds like chrome does.

**Comment by stevefaulkner** [comment]
Filed a bug on Firefox [implementation of input with datalist differs from HTML Acc API spec](https://bugzilla.mozilla.org/show_bug.cgi?id=1409084)
